packages/driver/src/cypress/cy.ts

```diff
@@ -363,6 +373,11 @@ export class $Cy extends EventEmitter2 implements ITimeouts, IStability, IAssert
   }
 
   fail (err, options: { async?: boolean } = {}) {
+    // if an onFail handler is provided, call this in its place (currently used for cross-origin support)
+    if (this.state('onFail')) {
```

Comment:
Typically this is the log associated with the command, correct?

Reply:
What's the log instance? `this.state('onFail')`?

This is only defined in the secondary driver and is used to catch any failures so they can be forwarded to the primary secondary.

Reply:
hmmm I think I'm confusing the onFail callback we pass to a log instance and the custom onFail we can attach to an error.